dnsproxy.cc.patch

```diff
@@ -0,0 +1,11 @@
+--- pdns/dnsproxy.cc.orig       Thu Oct 30 06:18:22 2014
++++ pdns/dnsproxy.cc    Mon Dec  8 14:55:41 2014
+@@ -169,7 +169,7 @@
+       memcpy(&d,buffer,sizeof(d));
+       {
+         Lock l(&d_lock);
+-#if BYTE_ORDER == BIG_ENDIAN
++#ifdef _LITTLE_ENDIAN
```

Comment:
Hi,

Where does the `_LITTLE_ENDIAN` definition come from, system sources? Why not use `#if BYTE_ORDER == LITTLE_ENDIAN` as defined (and used) in `dns.hh`?